zone/guild_mgr.cpp

```diff
@@ -561,10 +656,10 @@ void ZoneGuildManager::ProcessWorldPacket(ServerPacket *pack) {
 			}
 			LogGuilds("Processing ServerOP_OnlineGuildMembersResponse");
 			auto outapp = new EQApplicationPacket(OP_GuildMemberUpdate, sizeof(GuildMemberUpdate_Struct));
-			GuildMemberUpdate_Struct *gmus = (GuildMemberUpdate_Struct*)outapp->pBuffer;
+			GuildMemberUpdate_Struct* gmus = (GuildMemberUpdate_Struct*)outapp->pBuffer;
 			char Name[64];
 			gmus->LastSeen = time(nullptr);
-			gmus->InstanceID = 0;
+			gmus->InstanceID = 1;
```

Comment:
Why is this flipped ?

Reply:
I traced this to allow for correct display within the Guild Mgmt Window for online members